src/chatkit.ts

```diff
@@ -152,36 +209,50 @@ const TOKEN_EXPIRY_LEEWAY = 30;
 export default class Chatkit {
   apiInstance: Instance;
   authorizerInstance: Instance;
+  cursorsInstance: Instance;
   instanceLocator: string;
 
   private tokenWithExpiry?: TokenWithExpiryAt;
 
   constructor(options: Options) {
     const { instanceLocator, key, port, host, client } = options;
 
-    const apiInstanceOptions = ({
+    const sdkInfo = new SDKInfo({
+      productName: 'chatkit',
+      version: '0.13.0', // TODO extract from package.json
```

Comment:
Is this TODO relevant for now?

Reply:
Hmmm, let me see if I can address it quickly. If it takes too long I'll leave it.